src/react-motion.js

```diff
@@ -1,4 +1,7 @@
 /* @flow */
+
+export type { SpringHelperConfig } from './Types';
```

Comment:
Isn't it better to export all types? Sorry, I'm not using flow/ts, so cannot do much feedback on how it may affect downstream

Reply:
Some of types are internal. I think types are also a part of api. api with all possible types will be too verbose. I would export types with a request. This is the first one. Maybe the last one.